### DISCUSSION TITLE
Windows - Vanilla Setup

### DISCUSSION THREAD

**Discussion by giscus** [Uuvana-Studios/longvinter-server-documentation#2] [Announcements]
# Windows - Vanilla Setup

Longvinter Server Documentation for Server Administrators

https://docs-server.longvinter.com/windows/setup-vanilla/

**Comment by Khaztr** [comment]
Launching LongvinterServer.exe gives the following error: "Failed to open descriptor file ../../../Longvinter/Longvinter.uproject" and doesn't start.  Please advise.

**Reply by Khaztr** [reply]
disregard, I think I had an issue with GIT LFS, it's working now
